### ISSUE TITLE
Make ltap able to parse logfmt file

### ISSUE THREAD

**Issue by jonhoo** [brandur/hutils#16]
All the other binaries allow passing in a logfmt file. However, since `ltap` doesn't do this, there is currently no way of filtering a plain logfmt file before passing it through, say, `lfmt`.

**Comment by brandur** [comment]
Yep, great point. I'd originally intended `ltap` to be used more specifically as an API-driven source, so it's possible that this functionality might fit better as a different command that allows simple local filtering to be performed. Either way though, it certainly seems useful.
